packages/instant/src/components/buy_button.tsx

```diff
@@ -56,9 +57,9 @@ export class BuyButton extends React.Component<BuyButtonProps> {
         }
 
         let txHash: string | undefined;
+        const gasPrice = await gasPriceEstimator.getFastAmountInWeiAsync();
```

Comment:
Is there any way we can set the highest (as hard-coded value) gas price that we are willing to pay?